### PR TITLE
Add `--single-line` flag

### PR DESCRIPTION
**Pull request by corbinhesse**
Alias set to `l` as an `o` seemed synonymous with output. Another option would be `--minimal`; however, this word felt like it should remove colors and spinner.

### CODE REVIEW

**Comment by sindresorhus** [comment]
Why? What's the purpose?

**Comment by corbinhesse** [comment]
The output contains a single line of data and are four empty lines. Reducing the spacing allows to be run in other programs easily.

File: cli.js [hunk 1]
```diff
@@ -14,6 +14,7 @@ const cli = meow(`
 
 	Options
 	  --upload, -u  Measure upload speed in addition to download speed
+	  --oneline, -l  Reduce spacing and output on a single line
```
Comment (sindresorhus) [inline cli.js:17]: I would prefer `--minimal` as a flag name. I also don't see the point of a short-flag as this flag is not something you would type out directly, but instead have in a script.

Comment (sindresorhus) [reply cli.js:17]: You also need to update the readme output.

Comment (corbinhesse) [reply cli.js]: Resolved with new flag name.

File: cli.js [hunk 2]
```diff
@@ -14,6 +14,7 @@ const cli = meow(`
 
 	Options
 	  --upload, -u  Measure upload speed in addition to download speed
```
Comment (sindresorhus) [inline cli.js:16]:
```suggestion
	  --upload, -u   Measure upload speed in addition to download speed
```

File: cli.js [hunk 3]
```diff
@@ -24,21 +25,31 @@ const cli = meow(`
 		upload: {
 			type: 'boolean',
 			alias: 'u'
+		},
+		'single-line': {
```
Comment (sindresorhus) [inline cli.js:29]:
```suggestion
		singleLine: {
```

File: cli.js [hunk 4]
```diff
@@ -14,6 +14,7 @@ const cli = meow(`
 
 	Options
 	  --upload, -u  Measure upload speed in addition to download speed
+	  --single-line  Reduce spacing and output on a single line
```
Comment (sindresorhus) [inline cli.js:17]:
```suggestion
	  --single-line  Reduce spacing and output to a single line
```

**Pull request merged**